.travis.yml

```diff
@@ -1,15 +0,0 @@
-language:
-    python
-
-python:
-    - 3.6
-    - 3.6-dev
-
-install:
-    - pip3 install -r requirements.txt
-
-script:
-    pytest
-
-notifications:
-    email: false
```

Comment:
Travis was re-added and is now integrated into the development branch. Further tests are coming soon. :octocat: